CMakeLists.txt

```diff
@@ -243,7 +243,7 @@ endif()
 
 if(ENABLE_COVERAGE)
     include(AddLCov)
-    add_coverage("${CMAKE_BINARY_DIR}/run-tests.py")
+    add_coverage("${CMAKE_BINARY_DIR}/run-tests.py" "-*OutputBuffer")
```

Comment:
This might lead to turning off more than the desired test. Please use the full name :)